rbac/api/cross_access/view.py

```diff
@@ -82,8 +83,10 @@ class CrossAccountRequestViewSet(
     """
 
     permission_classes = (CrossAccountRequestAccessPermission,)
-    filter_backends = (filters.DjangoFilterBackend,)
+    filter_backends = (filters.DjangoFilterBackend, OrderingFilter)
     filterset_class = CrossAccountRequestFilter
+    ordering_fields = ("request_id", "start_date", "end_date", "created", "modified")
+    ordering = "-created"
```

Comment:
While I do see the argument of having this default to DESC ordering on `created` for implementation in the UI, I wonder if this would set a precedence that we don't yet have in the other API endpoints, which may lead to some confusion? As Zack mentioned in his initial Slack comment, as long as we allow `order_by=-created` (which would be the case here) that would be enough? We currently have default ordering on most APIs set alphabetically. I know that doesn't make sense here, but I feel like it would be less confusing to have more consistent defaults, allowing for custom sorting, vs adhoc defaults per endpoint?

Reply:
Yep, agree. Will remove the default